sopel/tools/target.py

```diff
@@ -61,33 +71,46 @@ class Channel:
     """A representation of a channel Sopel is in.
 
     :param name: the channel name
-    :type name: :class:`~.tools.Identifier`
+    :type name: :class:`~sopel.tools.identifiers.Identifier`
+    :param identifier_factory: A factory to create
+                               :class:`~sopel.tools.identifiers.Identifier`
```

Comment:
```suggestion
                               :class:`~sopel.tools.identifiers.Identifier`\\s
```

❔

Also I don't think this is the only place where such a docstring appears.

Reply:
I knew I should have wrote "instances of" and not bother with that `\\s`.